Seminar4/Task4.py

```diff
@@ -0,0 +1,33 @@
+# Задана натуральная степень k.
+# Сформировать случайным образом список коэффициентов (значения от 0 до 100)
+# многочлена и записать в файл многочлен степени k.
+# Пример: - k=2 => 2*x² + 4*x + 5 = 0 или x² + 5 = 0 или 10*x² = 0
+
+import random
+
+k = int(input("Введите натуральную степень k = "))
+lst = [random.randint(0, 101) for i in range(k + 1)]
```

Comment:
```suggestion
lst = [random.randint(0, 100) for i in range(k + 1)]
```
В задание указано от 0 до 100.